package.json

```diff
@@ -46,7 +46,7 @@
   "keywords": [
     "Scuttling"
   ],
-  "author": "JArnstein",
+  "author": "JArnstein (c) 2015",
```

Comment:
So in summary, 100% of the changes submitted here are wrong. Let me know when you've fixed it.